website/docs/Installation/linux-install.md

```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
 
 ## Installing Podman Desktop
 
```

Comment:
Should be nice to have a small sentence in the intro about the dependencies.
I think it's also where we should callout that podman is rootless.

Reply:
has been added in the latest commit